collection-renderer-lib/src/cmr/collection_renderer/api/routes.clj

```diff
@@ -1,9 +1,18 @@
 (ns cmr.collection-renderer.api.routes
   "Defines routes for fetching resources used in the Collection HTML"
-  (require [compojure.core :refer :all]
-           [cmr.common.services.errors :as errors]
-           [clojure.string :as str]
-           [clojure.java.io :as io]))
+  (:require
+   [clojure.java.io :as io]
+   [clojure.string :as str]
+   [cmr.common.services.errors :as errors]
+   [compojure.core :refer :all]))
+
+(def cmr-metadata-preview-gem
```

Comment:
As I just commented above. I am working with MMT to provide versioned gem so that we can configure different gem versions for different deployment environments. You are right. This will be in puppet config once we do that.